DEPENDENCIES.md

```diff
@@ -0,0 +1,41 @@
+| OpenShift GitOps              | 2.3.5                                                                                                                                 | Prerequisite for managing the installation and lifecycle of components       | OpenShift GitOps uses ArgoCD at the backend                                                                                                                               |
+| Postgres                      | 2.3.5                                                                                                                                 | Installed and setup as part of Tekton Results installation                   | For development purposes (store tekton results)                                                                                                                           | No requirement to use a particular version; users can install the latest version available at the time |
+| Amazon RDS                    | N/A                                                                                                                                   | External DB for Tekton Results                                               | For storing tekton results in an external database. No requirement to use a particular version. A specific configuration is required for connection and security purposes |
+
+
+### **Operators**
+
+| **Component**                | **Version**                            | **Purpose** | **Comments** |
+|------------------------------|----------------------------------------|-------------|--------------|
+| OpenShift Pipelines Operator | openshift-pipelines-operator-rh.v1.7.1 |             |              |
+| OpenShift Cert Manager       | openshift-cert-manager.v1.7.1          |             |              |
+| OpenShift GitOps Operator    | openshift-gitops-operator.v1.5.4       |             |              |
+
+### **Tools**
+
+| **Component**      | **Version**       | **Purpose**                                                                 | **Comments**                                                                                                                                                     |
+|--------------------|-------------------|-----------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| oc (OpenShift CLI) | 4.10              | To interact with the cluster                                                | Follows OpenShift version                                                                                                                                        |
```

Comment:
we use kubectl most of the time, which comes with oc but still could be listed. What do you think?

Reply:
Good catch. Let me add that.